src/module.c

```diff
@@ -3796,7 +3796,20 @@ void RM_FreeCallReply(RedisModuleCallReply *reply) {
     autoMemoryFreed(ctx,REDISMODULE_AM_REPLY,reply);
 }
 
-/* Return the reply type. */
+/* Return the reply type as one of the following:
+ * - REDISMODULE_REPLY_UNKNOWN
```

Comment:
IIRC you need a blank line before the bullet list for markdown to be rendered correctly.

Reply:
yes, indeed.. it screwed me up recently, here's the fixup commit as evidence: https://github.com/redis/redis-doc/commit/9877b67407317581b4381d82bd3572a095639936

Reply:
Thanks, fixed.